infra/command/format

```diff
@@ -157,6 +157,8 @@ function check_python_files() {
   FILES_TO_CHECK_PYTHON=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '\.py$'`)
   # Exceptional case: one-cmds don't have '.py' extension: ignore non-python source (cmake, etc) and ignore shell script: one-prepare-venv
   FILES_TO_CHECK_PYTHON+=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '^compiler/one-cmds/[^(\./)]*$' | egrep -v '^compiler/one-cmds/one-prepare-venv$'`)
+  # Exceptional case: onecc-docker don't have '.py' extension.
+  FILES_TO_CHECK_PYTHON+=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '^compiler/onecc-docker/[^(\./)]*$'`)
```

Comment:
Maybe `Dockerfile` should be ignored?

Reply:
`Dockerfile` is automatically ignored. Should I add it explicitly?

Reply:
I explicitly added the `onecc-docker` script so that other files are not included.